demo/index.html

```diff
@@ -77,9 +79,11 @@
       <div class="horizontal-section">
         <simple-menu multi>
           <a href="javascript:void(0)" role="menuitem">Item 0</a>
-          <a href="javascript:void(0)" role="menuitem">Item 1</a>
+          <a href="javascript:void(0)" role="menuitem" disabled>Item 1</a>
+          <a href="javascript:void(0)" role="menuitem" hidden>Ghost</a>
           <a href="javascript:void(0)" role="menuitem">Item 2</a>
           <a href="javascript:void(0)" role="menuitem">Item 3</a>
+          <a href="javascript:void(0)" role="menuitem" style="display:none">Another ghost</a>
```

Comment:
(this is in the demo, but i added it to the test proper. in the demo it looks sad because it's invisible but non-zero height)